### ISSUE TITLE
Installation on MacOS with nginx not possible with homebrew: invalid option --with-passenger

### ISSUE THREAD

**Issue by eikes** [phusion/passenger_library#109]
In this file: 

    passenger_library/source/install/install/step3.html.erb

it says `brew install nginx --with-passenger` to install nginx with precompiled passenger. But the [homebrew formula for nginx](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/nginx.rb) does not support this flag and brew returns:

    Error: invalid option: --with-passenger

**Comment by CamJN** [comment]
Yeah, the nginx module is now part of the passenger formula because homebrew removed all options from the built in formulae, I need to update the docs to say `brew install nginx passenger` will get you both nginx and passenger, and passenger comes with the nginx module.

**Comment by CamJN** [comment]
Hmm, it seems that `passenger_library/source/install/install/step3.html.erb` is not part of the most up to date docs at https://www.phusionpassenger.com/docs/tutorials/deploy_to_production/installations/oss/ownserver/ruby/nginx/. As you can see: https://github.com/phusion/passenger_library/tree/docs_2018/source/install/install/shared that file isn't used anymore.

**Comment by CamJN** [comment]
Yup, you'll see the notice at the top of the page noting that the new docs are at a different URL

**Comment by eikes** [comment]
I did eventually see that. My question is then why is there outdated documentation available? It seems this flag has not been available for years. But that is the page you get when you google for `nginx passenger macos`.
